packages/labs/cli/src/lib/commands/help.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import chalk from 'chalk';
+import commandLineUsage from 'command-line-usage';
+
+import {globalOptions} from '../options.js';
+import {Command, CommandOptions} from '../command.js';
+import {LitCli} from '../lit-cli.js';
+
+const CLI_TITLE = chalk.bold.underline('Lit CLI');
+// const CLI_DESCRIPTION = '...';
+const CLI_USAGE = 'Usage: `lit <command> [options ...]`';
```

Comment:
Maybe we don't want the backticks inside of this string? I feel like that's not so common.